ms2deepscore/data_generators.py

```diff
@@ -333,6 +335,17 @@ def _exclude_nans_from_labels(labels_df: pd.DataFrame):
             print(f"{n_dropped} nans among {len(labels_df)} labels will be excluded.")
         return clean_df
 
+    def _exclude_not_selected_inchikeys(self):
```

Comment:
It is not immediately clear from the method name and the docstring from *what* the inchikeys are removed (i.e. labels_df)

Reply:
True. I added `labels_df` as argument and extended the docstring.